applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_jump_process_2d.py

```diff
@@ -0,0 +1,55 @@
+import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+from KratosMultiphysics.CompressiblePotentialFlowApplication.define_wake_process_2d import DefineWakeProcess2D as DefineWakeProcess2D
+
+def Factory(settings, Model):
+    if(not isinstance(settings, KratosMultiphysics.Parameters)):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string")
+
+    return ComputeLiftJumpProcess2D(Model, settings["Parameters"])
+
+# TODO Implement this process in C++ and make it open mp parallel to save time selecting the wake elements
+class ComputeLiftJumpProcess2D(DefineWakeProcess2D):
```

Comment:
```suggestion
class ComputeLiftJumpProcess2D(KratosMultiphysics.Process):
```
This process doesn't need to inherit from DefineWakeProcess2D! You can simply inherit from the kratos Process class.